packages/react-aria-components/stories/GridList.stories.tsx

```diff
@@ -48,7 +48,8 @@ import './styles.css';
 
 export default {
   title: 'React Aria Components/GridList',
-  component: GridList
+  component: GridList,
+  excludeStories: ['MyGridListItem']
```

Comment:
In general, if it's just for tests, i prefer to copy it these days in the test file because i had to update tests or constrict stories based on the test